spec/services/provider_interface/reports/subject_rows_builder_service_spec.rb

```diff
@@ -11,6 +11,10 @@
   last_cycle: 'number_of_candidates_submitted_to_same_date_previous_cycle',
 }.freeze
 
+FIELD_MAPPING_THIS_CYCLE_ONLY = {
+  this_cycle: 'number_of_candidates_submitted_to_date',
+}.freeze
+
```

Comment:
This creates a constant at the top level in the whole test suite. Better to `stub_const` or use a `let` so the value is garbage collected